object/attribute.go

```diff
@@ -4,6 +4,22 @@ import (
 	"github.com/nspcc-dev/neofs-api-go/v2/object"
 )
 
+// Various system attributes.
+const (
+	// AttributeExpirationEpoch is a key to an object attribute that determines
+	// after what epoch the object becomes expired. Objects that do not have this
+	// attribute never expire.
+	//
+	// Reaction of NeoFS system components to the objects' 'expired' property may
+	// vary. For example, in the basic scenario, expired objects are auto-deleted
+	// from the storage. Detailed behavior can be found in the NeoFS Specification.
+	//
+	// Note that the value determines exactly the last epoch of the object's
+	// relevance: for example, with the value N, the object is relevant in epoch N
+	// and expired in any epoch starting from N+1.
+	AttributeExpirationEpoch = object.SysAttributeExpEpoch
```

Comment:
do we need "key", "system" words here?

Reply:
`key` is missing in existing ones, `system` - may be added, but i don't think user will care much and we'll have non-system expiration attribute

u think?

Reply:
i dont mind keeping it that way

Reply:
"key" is kinda obvious (in that it's not a value, but some attribute name/key). "system" as well, we don't have constants for random attributes here, we only have ones that have some meaning to the system.